core/src/main/java/com/linecorp/armeria/common/http/HttpRequestAggregator.java

```diff
@@ -46,7 +46,7 @@ public void onError(Throwable t) {
 
     @Override
     protected void doClear() {
-        trailingHeaders.clear();
+        trailingHeaders = null;
```

Comment:
I also thought about that and to think about consistency with HttpResponseAggregator, it was acceptable for me to assign null. (See [code](https://github.com/line/armeria/blob/f79467ad62eddd4b796fd25409cfdb63068831ac/core/src/main/java/com/linecorp/armeria/common/http/HttpResponseAggregator.java#L65))

Also, assigning HttpHeaders.EMPTY_HEADERS is also ok to me. So I fixed to use EMPTY_HEADERS.